components/ai_chat/core/common/features.cc

```diff
@@ -143,4 +143,12 @@ bool IsTabOrganizationEnabled() {
   return base::FeatureList::IsEnabled(features::kTabOrganization);
 }
 
+BASE_FEATURE(kNEARModels,
+             "AIChatNEARModels",
+             base::FEATURE_DISABLED_BY_DEFAULT);
+
+bool IsNEARModelsEnabled() {
+  return base::FeatureList::IsEnabled(features::kNEARModels);
+}
```

Comment:
I don't think we should be adding helper functions for simple IsEnabled checks like this

Reply:
In general, I agree, but I added it to be consistent with the features above